### PR TITLE
rollin fat doinks

### PR DESCRIPTION
**Pull request by actioninja**
## About The Pull Request
Getcha skins, getcha white owls

## Changelog
:cl:
add: smon,k weed
/:cl:

### CODE REVIEW

**Comment by HomocidlePancakes** [comment]
Dank.

**Comment by trollbreeder** [comment]
what

**Comment by Fikou** [comment]
damn brooooooooooooooooo, brooooooooo broo... good pr broooooooo

File: code/game/objects/items/cigs_lighters.dm [hunk 1]
```diff
@@ -367,6 +367,47 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 
 /obj/item/clothing/mask/cigarette/rollie/Initialize()
 	. = ..()
+	name = pick(list(
```
Comment (Qustinnus) [inline code/game/objects/items/cigs_lighters.dm:370]: this should be in a string file like other huge lists of texts. joint_names.txt 😏

**Comment by MMMiracles** [comment]
who the fuck calls it a bloint

**Comment by MrDoomBringer** [comment]
[thread theme](https://youtu.be/0xLi2RMzRfM)

File: code/game/objects/items/cigs_lighters.dm [hunk 2]
```diff
@@ -367,6 +367,47 @@ CIGARETTE PACKETS ARE IN FANCY.DM
...
+		"dutch",
+		"fatty",
+		"hogger",
+		"hooter",
+		"hootie",
+		"\improper J",
+		"jay",
+		"jimmy",
+		"joint",
+		"juju",
```
Comment (MrDoomBringer) [inline code/game/objects/items/cigs_lighters.dm:394]:
```suggestion
		"juju",
		"jeebie weebie",
```

File: code/game/objects/items/cigs_lighters.dm [hunk 3]
```diff
@@ -367,6 +367,47 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 
 /obj/item/clothing/mask/cigarette/rollie/Initialize()
 	. = ..()
+	name = pick(list(
+		"bifta",
+		"bifter",
+		"bird",
+		"blunt",
+		"bloint",
+		"boof",
```
Comment (RaveRadbury) [inline code/game/objects/items/cigs_lighters.dm:376]: Yes

**Comment by wesoda25** [comment]
over half of these names really piss me off

File: code/game/objects/items/cigs_lighters.dm [hunk 4]
```diff
@@ -367,6 +367,47 @@ CIGARETTE PACKETS ARE IN FANCY.DM
...
+		"jimmy",
+		"joint",
+		"juju",
+		"number",
+		"owl",
+		"phattie",
+		"puffer",
+		"reef",
+		"reefer",
+		"roach",
```
Comment (PKPenguin321) [inline code/game/objects/items/cigs_lighters.dm:401]: This is already in place for a burned out joint and shouldn't be on this list

**Comment by PKPenguin321** [comment]
>who the fuck calls it a bloint

The Bloods, who are well known for their longstanding rivalry with the Jips, refuse to say J sounds where possible. This goes to the point of spelling words like "joint" at "🅱oint," as a sort of censor.

File: code/game/objects/items/cigs_lighters.dm (hunk 1 above)
Comment (actioninja) [reply code/game/objects/items/cigs_lighters.dm:370]: no

**Comment by MMMiracles** [comment]
who knew ss13 would teach me street smarts

**Comment by PKPenguin321** [comment]
that was a joke

**Comment by MMMiracles** [comment]
you motherfucker I TRUSTED YOU

**Comment by PKPenguin321** [comment]
i mean the jips arent even a thing it was a play on the crips which is the real one

**Comment by Krysonism** [comment]
No "marihuana cigarette"?

File: code/game/objects/items/cigs_lighters.dm [hunk 5]
```diff
@@ -367,6 +367,47 @@ CIGARETTE PACKETS ARE IN FANCY.DM
...
+		"owl",
+		"phattie",
+		"puffer",
+		"reef",
+		"reefer",
+		"rollie",
+		"scoobie",
+		"shorty",
+		"spiff",
+		"spliff",
```
Comment (MrDoomBringer) [inline code/game/objects/items/cigs_lighters.dm:406]:
```suggestion
		"spliff",
		"sticky icky skunk skunk",
```

**Pull request merged**